person.rb

```diff
@@ -22,6 +24,10 @@ def correct_name
     @name
   end
 
+  def add_rental(date, person, book)
+    Rental.new(date, person, book)
+  end
```

Comment:
- [ ] [OPTIONAL] Kindly update this method to be like this:
```ruby
  def add_rental(book, date)
    Rental.new(date, self, book)
  end
```
You don't need to pass `person` as a parameter as you already have access to it on `self`.

Reply:
done